pages/common/git-shortlog.md

```diff
@@ -0,0 +1,19 @@
+# git shortlog
+
+> Summarizes the `git log` output.
+
+- View a summary of all the commits made, grouped alphabetically by author name:
+
+`git shortlog`
+
+- View a summary of all the commits made, sorted by the number of commits made:
+
+`git shortlog -n`
+
+- View a summary of all the commits made, grouped by the commiter identities:
```

Comment:
What does `commiter identities` mean ? Is it just name ?

Reply:
I couldn't even find this option in the man page. Can you paste the text here ?

Reply:
`-c / --committer`
>Collect and show committer identities instead of authors.

Reply:
Yes, it does seem to be email and name. Let's change the description to reflect that.

Reply:
@agnivade Done. I've kept "committer identities" in the description to make the man pages easy to understand if someone goes from TLDR to man